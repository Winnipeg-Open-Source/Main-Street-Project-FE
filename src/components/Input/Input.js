import React, { memo } from 'react';
import styled from 'styled-components';
import { Input as PCLNInput, getPaletteColor } from 'pcln-design-system';

const StyledInput = styled(PCLNInput)`
    background-color: ${getPaletteColor('background.lightest')};
`;

function Input (props) {
    return (
        <StyledInput {...props} />
    );
}

Input.displayName = 'Input';

export default memo(Input);
