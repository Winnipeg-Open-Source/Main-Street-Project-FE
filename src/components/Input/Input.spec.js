import React from 'react';
import renderWithTheme from 'tests/helpers/renderWithTheme';
import Input from './Input';

describe ('Input', () => {
    it ('renders without crashing', () => {
        const { asFragment } = renderWithTheme(<Input />);
        expect(asFragment()).toMatchSnapshot();
    });
});
