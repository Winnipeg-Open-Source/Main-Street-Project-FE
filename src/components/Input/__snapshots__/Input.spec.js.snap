// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pkfPC haHNKR sc-psDhf iAENMo"
  >
    <input
      class="sc-qcrrk sc-qQxFs iUvCRq"
    />
  </div>
</DocumentFragment>
`;
